scripts/select_appointments.sql

```diff
@@ -1 +1,9 @@
 -- retrieve appointment information
+
+SELECT p.family_name, a.date, a.time
```

Comment:
This won't run because you named your columns `start_date` and `start_time`, but you're trying to access the `date` and `time` columns, which don't exist.  If the column names were consistent, then this would run without a problem and produce the correct result for the first part of the question.

```sql
SELECT p.family_name, p.given_name, a.time
FROM appointments AS a
INNER JOIN patients AS p
  ON a.patient_id = p.id
INNER JOIN doctors AS d
  ON a.doctor_id = d.id
WHERE d.family_name = 'Hubbard';

SELECT d.family_name, d.given_name, a.time
FROM appointments AS a
INNER JOIN doctors AS d
  ON a.doctor_id = d.id
INNER JOIN patients p
  ON a.patient_id = p.id
WHERE p.family_name = 'Wilcox' AND p.given_name = 'Marsha';
```